app/views/users/_past.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="p-4">
+<% if @past_events.empty? %>
+  <%= "There're no previous events" %>
+<% else %>
+<% @past_events.each do |inv| %>
+<div class="card" style="width: 18rem;">
+  <div class="card-body">
+  <h5 class="card-title"><%= "#{@user.name} was invited to an event in #{inv.location } " %> </h5>
+  <p class="card-text"> <%= "Event date: #{inv.date.to_formatted_s(:rfc822)}." %></p>
+  <%= link_to 'Check event', event_show_path(inv)  %>
+  </div>
+</div>
+<% end %>
+<% end %>
```

Comment:
- Move the if/else statement to a helper method that you call here and render the markup within the helper method using a partial